manifests/init.pp

```diff
@@ -315,6 +315,11 @@
 #   Whether we should test new configuration files with `corosync -t`.
 #   (requires corosync 2.3.4)
 #
+# @param watchdog_device
+#   Watchdog device to use, for example '/dev/watchdog' or 'off'.
+#   Its presence (or lack thereof) shifted with corosync versions.
+#   Default: undef
```

Comment:
Nit: we don't document defaults since puppet-strings can figure that out already
```suggestion
```